collections/e12c_motorway.yaml

```diff
@@ -1,4 +1,5 @@
 Name: E12C_Motorway
+OpenEOPID: E12C_Motorway
```

Comment:
My mistake, names should always be all caps so it should be `E12C_MOTORWAY`. Same goes for `E12D_PRIMARY`.

Reply:
Done!

Reply:
Thanks!